Todo-Redux/webpack.config.development.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+var path = require('path');
```

Comment:
here also you can use import instead of require, so es6 styling will be maintained throughout it project.